### PR TITLE
docs(js): Create readme for js subfolder

### LINKED ISSUE #1814: Add documentation in react-native app subfolder root
Need at least minimal Up & Running instructions for developers in ./js

- [ ] Software dependency list, at minimum for macOS (xcode, CocoaPods, ...) and installation instructions
- [ ] Makefile usage (abstractions clarified)
- [ ] ...?

Expands subtask of #1754

### PR DESCRIPTION
**Pull request by ekelen**
Instructions for installing and running debug and storybook dev environments.

Closes #1814

### CODE REVIEW

File: js/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#### Requirements for iOS
+
+* OS X
+* Xcode [TODO: version?]
+* Node & Watchman (`brew install node`, `brew install watchman`)
+* CocoaPods (`sudo gem install cocoapods` or follow official [sudo-less instructions](https://guides.cocoapods.org/using/getting-started.html#sudo-less-installation))
+
+#### Requirements for Android
+
+* [TODO]
```
Comment (clegirar) [inline js/README.md:25]: Do you want to list the requirements to build the app for android in this PR ?

Comment (ekelen) [reply js/README.md:25]: Happy to 👍 I'll re-request review when added

Comment (ekelen) [reply js/README.md]: They are added @clegirar !

File: js/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+* An installation of [golang](https://golang.org/dl/)
+* Your `$GOPATH` explicitly set to your golang directory (usually `$HOME/go`)
+* Java 8. If you already have another version of Java, you can use a version manager and Homebrew to add another installation. Some nice instructions are given [here](https://java.christmas/2019/16).
+* The gomobile package: `go get golang.org/x/mobile/cmd/gomobile/... && gomobile init`
+
+### Build for iOS
+
+`make start.metro`
+
```
[the remark this answers is not in the document]
Comment (ekelen) [reply js/README.md:24]: Done

File: js/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+`make run.storybook.ios`
+
+#### Android
+
+`make run.storybook.android`
+
+## Known issues
+
+- `make run.storybook.*` outputs error `Error: => Create a storybook config file in "./.storybook/config.{ext}".` during build
+- GRPC errors on iOS and Android views
```
Comment (moul) [inline js/README.md:76]:
```suggestion
- gRPC errors on iOS and Android views
```

File: js/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+The installation list may look daunting, but if you have some experience in mobile development, you likely have most of them already!
+
+In addition to the _General requirements for React Native_ below, you'll need:
+
+- An installation of [golang](https://golang.org/dl/)
+- Your `$GOPATH` explicitly set to your golang directory (usually `$HOME/go`)
+- Java 8. If you already have another version of Java, you can use a version manager and Homebrew to add another installation. Some nice instructions are given [here](https://java.christmas/2019/16).
+- The gomobile package: `go get golang.org/x/mobile/cmd/gomobile/... && gomobile init`
+- [Bazel](https://docs.bazel.build/versions/master/install-os-x.html#install-with-installer-mac-os-x)
```
Comment (moul) [inline js/README.md:19]: is it really mandatory now for mobile dev on Berty?

File: js/README.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+### 1. Requirements
+
+The installation list may look daunting, but if you have some experience in mobile development, you likely have most of them already!
+
+In addition to the _General requirements for React Native_ below, you'll need:
+
+- An installation of [golang](https://golang.org/dl/)
+- Your `$GOPATH` explicitly set to your golang directory (usually `$HOME/go`)
+- Java 8. If you already have another version of Java, you can use a version manager and Homebrew to add another installation. Some nice instructions are given [here](https://java.christmas/2019/16).
```
Comment (moul) [inline js/README.md:17]: if only needed for Android, specify it

File: js/README.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+## General requirements for React Native
+
+- `yarn` to install Nodejs dependencies
+- Homebrew or package manager of choice
+
+### Requirements for iOS
+
+- OS X
+- Xcode [TODO: version?]
```
Comment (moul) [inline js/README.md:45]:
```suggestion
- Xcode _(latest stable)_
```

File: js/README.md (hunk 4 above)
Comment (ekelen) [reply js/README.md:19]: Do you mean having these instructions in general, or a specific item? I wasn't able to run `*.debug` without all of these items.

Comment (moul) [reply js/README.md:19]: I was asking specifically about bazel

ok, so mandatory :+1:, keep it here

File: js/README.md (hunk 6 above)
Comment (ekelen) [reply js/README.md]: Updated

File: js/README.md (hunk 3 above)
Comment (ekelen) [reply js/README.md]: Done

**Pull request merged**